source/grid/manifold.cc

```diff
@@ -423,7 +423,8 @@ get_new_point (const Quadrature<spacedim> &quad) const
     if (periodicity[d] > 0)
       for (unsigned int i=0; i<surrounding_points.size(); ++i)
         {
-          minP[d] = std::min(minP[d], surrounding_points[i][d]);
+          if (weights[i] > tolerance)
+            minP[d] = std::min(minP[d], surrounding_points[i][d]);
```

Comment:
ah, ok, this is where this change now went. this should also fix the currently failing tests I think.